content/en/user/posting.md

```diff
@@ -47,9 +47,9 @@ You can attach either files or a poll to your status.
 
 Click the paper clip to attach a file to your post. You can attach the following:
 
-* **Images** (PNG, JPG, GIF) **up to 8MB**. Images will be downscaled to 1.6 megapixels (enough for a 1280x1280 image). Up to 4 images can be attached.
+* **Images** (PNG, JPG, GIF) **up to 16MB**. Images will be downscaled to 8.2 megapixels (enough for a 4k (3840x2160) image). Up to 4 images can be attached.
```

Comment:
i'd take out the "4k" mention but otherwise lgtm

Reply:
with some possible consideration for how to express version history, if any. maybe it's good enough to just list the latest limits?

Reply:
Jeg mener, at den nyeste version er tilstrækkelig, og for ældre versioner bør du forvente, at de opgraderer til den seneste stabile version.